.gitignore

```diff
@@ -6,6 +6,7 @@ Makefile.in
 _*
 .deps/
 .libs/
+.vscode/
```

Comment:
this should go in your global .gitignore

Reply:
fixed